faqs/galaxy/account_update_preference.md

```diff
@@ -8,19 +8,19 @@ contributors: [jennaj, bernandez]
 
 
 1. Log in to Galaxy
-2. Navigate to **User > Preferences** on the top menu bar.
+2. Navigate to **User -> Preferences** on the top menu bar
```

Comment:
```suggestion
2. Navigate to [**User -> Preferences**](https://my.galaxy.training/?path=/user) on the top menu bar
```

We've setup the 'my.galaxy.training' service to allow redirecting to the user's preferred galaxy, it let's the enter the domain and then go directly to their galaxy, with galaxy-independent URLs (like to /user). It doesn't work for e.g. workflows given their different IDs but it might be nice to use it in this one place, in case the user can't find it